tests/cases/rest_decorator_test.py

```diff
@@ -94,3 +94,7 @@ def testRawResponse(self):
         resp = self.request('/other/rawInternal', isJson=False)
         self.assertStatusOk(resp)
         self.assertEqual(self.getBody(resp), 'this is also a raw response')
+
+        resp = self.request('/other/rawReturningText', isJson=False)
+        self.assertStatusOk(resp)
+        self.assertEqual(self.getBody(resp), 'this is not encoded')
```

Comment:
We want to test this with actual calls, so that we'd see the failure condition in the test.  The test has to specify BIND_SERVER in the CMakeLists file and set a port at the top.

The actual test would be something like:
```
import requests

def testEndpointUnicode(self):
    req = requests.get(
            'http://127.0.0.1:%d/api/v1/other/rawReturningText' % (
                int(os.environ['GIRDER_PORT'])))
        self.assertEqual(req.status_code, 200)
        self.assertEqual(req.text, 'this is not encoded')
```

I'd also add an upper code point character in the string to verify that it comes through as UTF-8 (say `\U0001F44D` and come through as `\xf0\x9f\x91\x8d`.